topgun/k8s/baggageclaim_drivers_test.go

```diff
@@ -46,15 +49,21 @@ var _ = Describe("baggageclaim drivers", func() {
 			It("successfully recreates the worker", func() {
 				By("deploying concourse with ONLY one worker and having the worker pod use the gcloud disk and format it with btrfs")
 
+				scName := "btrfs"
+				createBtrfsStorageClass(scName)
```

Comment:
not sure about this change. 
- from one perspective, it allows to create the storage class if it is not found.
- but from a security perspective, in order to achieve that the service account that is used should have admin privileges on the cluster, I am not sure about the one that's used now in the test, but if it doesn't have enough privileges, and I think it shouldn't, then the test will fail.

as an alternative, I am thinking maybe we can only check if the storage class doesn't exist then we can either fail the test or skip it 🤔

Reply:
Fair point regarding the security  aspect. Our tests rely on being able to create and manage namespaces already. 
What would be the recommended way of managing  the storage class so that we minimize manual intervention when someone targets any vanilla cluster ?

Reply:
I am not super familiar with the roles and privileges of accounts, if creating a namespace requires the same privileges as creating a storage class, then for sure we can have the test create the storage class if it is not found.

Reply:
I am assuming a vanilla cluster wouldn't have the storage class, so the test wouldn't be runnable, thus we can skip the tests that depend on such storage class, maybe 🤔

Reply:
I looked at it briefly and it should be possible to provide much more [granular permissions](https://kubernetes.io/docs/reference/access-authn-authz/rbac/) and its probably a good idea to have more restricted service accounts. Currently, our service accounts have carte blanche access and this PR doesn't extend that. We'd be able to provide the ability to a service account to just create/manage storageclass without giving it other admin privileges.  Should we track/triage that as a separate story as that has a different objective of cleaning up permissions?

Reply:
sure .. sounds good to me